go.mod

```diff
@@ -1,9 +1,12 @@
 module github.com/bwplotka/bingo
 
-go 1.14
+go 1.15
```

Comment:
Do you actually use any features specific to go 1.15?

Reply:
Not really good point